tests/match_view.rs

```diff
@@ -139,17 +141,17 @@ fn test_match_view_two_variants_invalid_layout() {
     let font = CharacterBufferFont;
     let make_view = |state| {
         match_view!(state => {
-            0 => Text::str("zero\n!!!", &font),
+            0i32 => Text::str("zero\n!!!", &font),
```

Comment:
Undecided on this one

Reply:
yeah, it looks weird, a solution would be to inject the `#[allow()]` attribute inside the macro